modules/react/tooltip/lib/OverflowTooltip.tsx

```diff
@@ -41,7 +41,8 @@ export const findEllipsisElement = (element: Element): Element | null => {
   if (style.textOverflow === 'ellipsis' || Number(style.webkitLineClamp) > 0) {
     return element;
   } else if (element.children) {
-    // `children` is not defined for SVGElement in IE11
+    // `children` is not defined for SVGElement in IE11 Do we still need this?
+
```

Comment:
So I tried to remove this but that broke, any ideas if this has to be rewritten or kept?

Reply:
I don't fully remember the context, but I think it was the difference of the if statement:

```ts
} else {
```

I think that gave an error in IE because `element.children` is not defined for SVG elements in IE, so this `else` became a type guard instead. The type definitions say `children` is always available on all `Element` interfaces (`SVGElement` extends `Element`). The `} else if (element.children) {` simply guarded against IE failing when an SVG element was encountered because SVG elements in IE don't properly extend `Element`s